modules/setting/incoming_email.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package setting
+
+import (
+	"fmt"
+	"net/mail"
+	"strings"
+
+	"code.gitea.io/gitea/modules/log"
+)
+
+var IncomingEmail = struct {
```

Comment:
don't forgot update the [`custom/conf/app.example.ini`](https://github.com/go-gitea/gitea/blob/main/custom/conf/app.example.ini)